src/tokens/transformation/ios/colorsets.js

```diff
@@ -45,8 +45,16 @@ module.exports = {
     dictionary.allTokens
       .filter((token) => token.type === 'color')
       .forEach((token) => {
-        const colorsetPath = `${assetPath}/${changeCase.pascalCase(
-          token.path.slice(2).join(' ')
+        if (token.name === 'ColorBlack' || token.name === 'ColorWhite') {
+          // The system already provides these and will conflict with the names
+          return
+        }
+        const colorsetPath = `${assetPath}/${changeCase.snakeCase(
+          token.path
+            .filter((i) => {
```

Comment:
Optional: 
```suggestion
            .filter((i) => i !== 'dark' && i !== 'light' && i !== 'color')
```

we should change the `!=` to `!==` though, even if you don't move to an inline arrow function

Reply:
Fixed! Though the formatter forced it to include the parenthesis for the arg and the conditions onto a separate line